github_plugin/sync.py

```diff
@@ -14,52 +13,53 @@
 from datetime import datetime
 from dateutil import parser
 import time
-from external_plugins.zendesk_plugin import transformer_functions
+import json
+from external_plugins.github_integrate import transformer_functions
```

Comment:
change the github_integrate  to github_plugin

Reply:
done